src/events.rs

```diff
@@ -24,7 +24,7 @@ pub struct EventPrepare {
 }
 
 impl EventPrepare {
-    /// get_content returns the content of the event to be signed
+    /// //get_content returns the content of the event to be signed
```

Comment:
Why did you put a comment here ?

Reply:
I thought I got a clippy error to do that, but I just removed it and ran clippy locally and its not needed

Reply:
Just pushed a fix, Sorry about the extra CI runs